lib/tags.js

```diff
@@ -87,19 +88,25 @@ exports.parseTagLine = (line) => {
  * console.log(footer.Bar);
  * > baz
  */
-exports.parseFooterTagLines = (footerTagLines) => {
+exports.parseFooterTagLines = (footerTagLines, lowerCaseFooterTags) => {
   return _.chain(footerTagLines)
     .reject((line) => {
       return _.isEmpty(line.trim());
     })
     .map((tagLine) => {
       const tag = exports.parseTagLine(tagLine);
+      const finalTags = [];
 
-      return [
-        tag.key,
-        tag.value
-      ];
+      finalTags.push([ tag.key, tag.value ]);
```

Comment:
So in that case a `map` no longer makes sense, since the intention is to return either one item or two per tag line. What about something like this?

```js
exports.parseFooterTagLines = (footerTagLines, lowerCaseFooterTags) => {
  return _.chain(footerTagLines)
    .reject((line) => {
      return _.isEmpty(line.trim());
    })
    .reduce((tags, tagLine) => {
      const tag = exports.parseTagLine(tagLine);

      tags.push([
        tag.key,
        tag.value
      ]);

      if (lowerCaseFooterTags) {
        tags.push([
          tag.key.toLowerCase()
          tag.value
        ]);
      }

      return tags;
    }, [])
    .fromPairs()
    .value();
};
```

There is no need to check for duplicates since `_.fromPairs()` will take care of that automatically:

```js
> const lodash = require('lodash');
> lodash.fromPairs([ [ 'a', 1 ], [ 'a', 1 ] ]);
{ a: 1 }
```